test/create-config.test.ts

```diff
@@ -0,0 +1,33 @@
+import { join } from 'path';
+import prompts from 'prompts';
+import { OperationsRepository } from '../src/types';
+import { readOperationsRepository } from '../src/utils/read-operations';
+import { createConfig } from '../src/create-config';
+import { writeOperationsRepository } from '../src/utils/write-operations';
+
+describe('create-config', () => {
+  let originalMockOpsRepo: OperationsRepository;
+
+  beforeAll(() => {
+    originalMockOpsRepo = readOperationsRepository(join(__dirname, 'fixtures', 'data'));
+  });
+
+  it('builds the airnode and airkeeper configs', async () => {
+    const date = new Date().toISOString().split('T')[0];
+
+    prompts.inject(['api3', false, false]);
+    await createConfig(join(__dirname, 'fixtures', 'data'));
+
+    const newMockOpsRepo = readOperationsRepository(join(__dirname, 'fixtures', 'data'));
+    expect(newMockOpsRepo.apis.api3.deployments[date].airnode).toEqual(
+      originalMockOpsRepo.apis.api3.deployments['2022-04-17'].airnode
+    );
+
+    expect(newMockOpsRepo.apis.api3.deployments[date].airkeeper).toEqual(
+      originalMockOpsRepo.apis.api3.deployments['2022-04-17'].airkeeper
+    );
```

Comment:
I have to admit I'm confused about these expects. Does `createConfig()` writes files under `deployments/`?

Reply:
yes, `createConfig()` creates a new deployment under `/deployments` with the current date. This new configuration with all the files `(config,json, secrets.env)`. This new deployment should match the already pre configured deployment(which we know is correct).